### PR TITLE
added multiply function

### PR DESCRIPTION
**Pull request by rolandpinter**

### CODE REVIEW

File: proba.py [hunk 1]
```diff
@@ -1,5 +1,8 @@
 def add(a, b):
     return a+b
 
+def multiply(a, b):
+    return a*b
```
Comment (JavorErv) [inline proba.py]: add comment (#)

File: proba.py [hunk 2]
```diff
@@ -1,5 +1,8 @@
 def add(a, b):
     return a+b
 
+def multiply(a, b):
```
Comment (JavorErv) [inline proba.py]: type hint

**Review by Tekjesz — CHANGES_REQUESTED** [review]

File: proba.py (hunk 2 above)
Comment (rolandpinter) [reply proba.py]: done

File: proba.py (hunk 1 above)
Comment (rolandpinter) [reply proba.py]: done

**Review by Tekjesz — APPROVED** [review]

**Pull request merged**